nbs/core.ipynb

```diff
@@ -963,13 +963,19 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #694.            _, forefront_offset = np.divmod((series_length - test_size - self.h), step_size)
</pre><p>These are all ints, right? So I think this could just be <code>forefront_offset = (series_length - test_size - self.h) % step_size</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/933/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>